setup.cfg

```diff
@@ -45,10 +45,6 @@ classifiers =
 [options]
 zip_safe = False
 python_requires = >=3.6
-install_requires = setuptools
-setup_requires =
-    setuptools
-    build
```

Comment:
Should this ('build')  be added to pyproject.toml's 

```
[build-system]
build-backend = "setuptools.build_meta"
requires = ["setuptools"]
```

Reply:
No, it is a frontend. It is not used on the backend because it's `build` insalling stuff from `requires`.